app/views/admin/alert_types/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Edit: <%= @alert_type.title %> alert type</h1>
+
+<%= simple_form_for(@alert_type, url: admin_alert_type_path(@alert_type)) do |f| %>
+
+  <%= render 'shared/errors', subject: @alert_type, subject_name: 'alert type' %>
+
+  <div class="form-group">
```

Comment:
Do we need these wrapper divs with simple_form?

Reply:
*cough* no.